pkg/types/csv_rbac_test.go

```diff
@@ -0,0 +1,190 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFilterRules(t *testing.T) {
```

Comment:
Would favor `operator-framework/api` types, but can still wrap them in the custom `CsvPermissions` wrapper type. Maybe rename to `CSVPermissions`, feels this would follow go naming philosohpy more closely.